utilities/src/main/java/io/syndesis/qe/resource/impl/Kudu.java

```diff
@@ -21,26 +22,76 @@
 @Slf4j
 public class Kudu implements Resource {
     private static final String APP_NAME = "syndesis-kudu";
-    private static final String API_APP_NAME = "kudu-rest-api";
-    private static final String ROUTE_NAME = "kudu";
     private static final String LABEL_NAME = "app";
+    private static final String ROUTE_NAME = "kudu";
+    private static final String API_APP_NAME = "kudu-rest-api";
+    public static final int KUDU_PORT = 7051;
+    public static final int REST_PORT = 8080;
 
     @Override
     public void deploy() {
         if (!TestUtils.isDcDeployed(APP_NAME)) {
-            //OCP4HACK - openshift-client 4.3.0 isn't supported with OCP4 and can't create/delete templates, following line can be removed later
-            OpenShiftUtils.binary()
-                .execute("create", "-f", Paths.get("../utilities/src/main/resources/templates/syndesis-kudu.yml").toAbsolutePath().toString());
-            //            try (InputStream is = ClassLoader.getSystemResourceAsStream("templates/syndesis-kudu.yml")) {
-            //                OpenShiftUtils.getInstance().load(is).createOrReplace();
-            //            } catch (IOException e) {
-            //                fail("Template processing failed", e);
-            //            }
+            List<ContainerPort> ports = new LinkedList<>();
+            ports.add(new ContainerPortBuilder()
+                .withName(APP_NAME)
+                .withContainerPort(KUDU_PORT)
+                .withProtocol("TCP").build());
+
+            List<EnvVar> templateParams = new ArrayList<>();
+            OpenShiftUtils.getInstance().deploymentConfigs().createOrReplaceWithNew()
+                .editOrNewMetadata()
+                .withName(APP_NAME)
+                .addToLabels(LABEL_NAME, APP_NAME)
+                .endMetadata()
+
+                .editOrNewSpec()
+                .addToSelector(LABEL_NAME, APP_NAME)
+                .withReplicas(1)
+                .editOrNewTemplate()
+                .editOrNewMetadata()
+                .addToLabels(LABEL_NAME, APP_NAME)
+                .endMetadata()
+                .editOrNewSpec()
+                .addNewContainer().withName(APP_NAME).withImage("syndesisqe/kudu-2in1:latest")
```

Comment:
where are the dockerfiles for these containers located?

Reply:
here: git clone git@github.com:lgarciaaco/kudu-docker.git

Reply:
//1. clone kudu-docker project:
git clone git@github.com:lgarciaaco/kudu-docker.git

//2. in kudu-docker project, docker-entrypoint.sh, add to KUDU_TSERVER_OPTS= these 2 lines:
`   -memory_limit_hard_bytes=7294967296 \
   -block_cache_capacity_mb=1024 \`

//3. change ENTRYPOINT in kudu-docker, Dockerfile:
`ENTRYPOINT ["/docker-entrypoint.sh" , "single"]`

Reply:
kudu rest api 'how to do' is here:
git@github.com:mcada/kudu-rest-app.git